### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility][Jaws/NVDA][MAS4.2.1][IntelliJ][Wipro]:Jaws and NVDA announces additional inappropriate information when "Create service principle Status" dialog is opened 

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1478]
**Repro Steps:**

1. Open and Start IntelliJ.
2. Navigate to Tools menu-> "Azure"-> "Azure Sign In"
3. In "Azure Sign in" dialog, tab to and select "Automated" radio button.
4. Tab to and select "Browse" button.
5.  In "Create Authentication Files" dialog , tab to and select required subscription and select start button.
6. "Create service principle Status" dialog will open
7. Observe the behaviour of Jaws and NVDA

**Actual Result:**
Jaws and NVDA announces additional inappropriate information when "Create service principle Status" dialog  is opened 

**Jaws behaviour:**
  untitled4 - IntelliJ IDEA frame
Create Service Principal Status dialog
Step Result Details Created authentication file(s) (one per Active Directory instance/tenant): 
OK Button
To activate press spacebar.

**NVDA Behaviour:**
Create Service Principal Status  dialog  Created authentication file(s) (one per Active Directory instance/tenant):
OK  button

**Expected Result:**
Jaws and NVDA should announce only the dialog name when dialog is opened .
i.e Create Service Principal Status dialog

**Platform:**
IntelliJ IDEA 2017.3.2 (Community edition)
Build# IC-173.4127.27
JRE:1.8.0_152-release-1024-b8amd64
JVM: OpenJDK 64 Bit Server VM by Jetbrains.
OSBuild Version: Version 1709(Os Build 16299.192)
![cretae service principle status](https://user-images.githubusercontent.com/22956130/36105018-94e6f2ea-1039-11e8-8b55-6c3e55abfcf2.jpg)
![cretae service-nvda](https://user-images.githubusercontent.com/22956130/36105021-98e0b070-1039-11e8-9a4d-8351ce92789d.jpg)

**Comment by Eskibear** [comment]
Cannot reproduce it.

According to the practice, NVDA cannot read content of any dialog in IntelliJ IDEA 2017.3.4 Community.
A ticket is created to track the issue.

Ref:
https://youtrack.jetbrains.com/issue/IDEA-191325

**Comment by sallyur** [comment]
As mentioned above, in our practice we cannot reproduce the issue, as NVDA cannot read content of IntelliJ IDEA at all.

- Importance of experience: it depends, varies from somewhat important to important.
- Quality of experience: not impacted. Since the Narrator is not experiencing this kind of issues, users can use the Narrator to as an alternative workaround.
- Alternate solutions: No alternate technologies or options can be leveraged.
- Evidence: All evidence is provided in this issue and external issue.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749913
